tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/DevOpsService.cs

```diff
@@ -89,7 +89,7 @@ public interface IDevOpsService
         public Task<ReleasePlanDetails> GetReleasePlanForWorkItemAsync(int workItemId);
         public Task<ReleasePlanDetails> GetReleasePlanAsync(string pullRequestUrl);
         public Task<WorkItem> CreateReleasePlanWorkItemAsync(ReleasePlanDetails releasePlan);
-        public Task<Build> RunSDKGenerationPipelineAsync(string branchRef, string typespecProjectRoot, string apiVersion, string sdkReleaseType, string language, int workItemId);
+        public Task<Build> RunSDKGenerationPipelineAsync(string branchRef, string typespecProjectRoot, string apiVersion, string sdkReleaseType, string language, int workItemId, string sdkRepoBranch = "main");
```

Comment:
sdkRepoBranch is the source branch with SDK changes. We should not default this to main. Default should be empty.